trivy_report/report_issues.py

```diff
@@ -76,6 +76,9 @@ def main():
         reports = parse_results(data, existing_issues=existing_issues)
     except TypeError as e:
         abort(f"Failed to parse Trivy JSON report: {e}")
+    except KeyError as e:
```

Comment:
As mentioned above, make parse_results() return False or None and then handle that instead.